addons/mail/static/src/core/common/messaging_service.js

```diff
@@ -25,32 +23,27 @@ export class Messaging {
         this.userSettingsService = services["mail.user_settings"];
         this.router = services.router;
         this.isReady = new Deferred();
-        this.imStatusService = services.im_status;
         const user = services.user;
         this.store.Persona.insert({ id: user.partnerId, type: "partner", isAdmin: user.isAdmin });
-        this.registeredImStatusPartners = reactive([], () => this.updateImStatusRegistration());
-        this.store.registeredImStatusPartners = this.registeredImStatusPartners;
+        /** @deprecated */
+        this.registeredImStatusPartners = [];
```

Comment:
will it still contain the correct value in stable?

Reply:
Yes I added `onAdd()` and `onDelete()` in `imStatusTrackedPersonas` relational field